mlflow/store/artifact/dbfs_artifact_repo.py

```diff
@@ -183,6 +189,7 @@ def dbfs_artifact_repo_factory(artifact_uri):
             and not artifact_uri.startswith("dbfs:/databricks/mlflow-registry"):
         # If the DBFS FUSE mount is available, write artifacts directly to /dbfs/... using
         # local filesystem APIs
+        # TODO(sueann): remove databricks profile info
```

Comment:
also, check that the host_uri is not default -> use `DbfsRestArtifactRepository`